xdsl/parser.py

```diff
@@ -136,8 +137,6 @@ def get_char(self,
             return None
         return self.str[self._pos.idx:self._pos.idx + n]
 
-    _T = TypeVar("_T")
```

Comment:
Should we really get the `_T` from utils here?
What if we change the constraints we have on `_T` in `util`?
I thought that starting a variable with an underscore is making it private for the file?

Note that I'm not sure what is the standard way of doing this in Python, so please ignore if this is how it is supposed to be done.

Reply:
I will leave it as before, cuz I am not confident for other way